### DISCUSSION TITLE
Does someone know if I can upscale an existing image using the hires fix settings?

### DISCUSSION THREAD

**Discussion by yishayraziel** [lllyasviel/stable-diffusion-webui-forge#1933] [General]
Does someone know if I can upscale an existing image using the hires fix settings?

**Comment by HMRMike** [comment]
Hires fix is basically img2img. Just use img2img and remember to enable an upscaler choice for it, like you have in hires fix.

**Comment by yishayraziel** [comment]
Thanks!!!


Yishay Raziel

Cell: 0584-902-902
Tel: 0774-902-902
Fax: 053-955-7546
Email: ***@***.***
Site: www.yraziel.com

**Comment by yishayraziel** [comment]
Hi it’s me again

When I’m using img2img and enabling the hires its changes the image
completely.
What am I doing wrong?

**Comment by HMRMike** [comment]
Usually that's because of the Denoise setting being way too high. Also if the source image is too blurry or low quality, the upscaled version will need more denoise, and thus more changes. 

You can also try adding controlnet and use tile mode, a good model for that would be Union- it can work well with low quality input (you can also use the preprocessor to blur the image a bit, I find it helps sometimes). The result will be more faithful to the original image, but the settings may need a lot of trial and error. 
https://huggingface.co/xinsir/controlnet-union-sdxl-1.0

**Comment by yishayraziel** [comment]
Thanks, I'll check it out.
The thing is,
After creating an image from text and using the upscale button (using
default hires settings) I’m getting an amazing faithful upscale, that I’m
not able to get using img2img

Does it make sense?

**Comment by HMRMike** [comment]
It should definitely be possible to replicate hires fix results through img2img. Although it's not always 1:1 in some cases, I'm still not sure exactly why. 
Maybe these examples here could help: https://github.com/lllyasviel/stable-diffusion-webui-forge/discussions/758#discussioncomment-10514428
